src/movableCoord.js

```diff
@@ -171,14 +173,14 @@ eg.module("movableCoord", ["jQuery", eg, "Hammer"], function($, ns, HM) {
 			} catch (e) {}
 		},
 
-		_convertInputType: function(inputType) {
+		_convertInputType: function(inputType, options) {
```

Comment:
I think option is unnecessary.
string or boolean type is enough.

could you use boolean or string type?